packages/mc-scripts/src/config/optimizations.ts

```diff
@@ -0,0 +1,66 @@
+import chalk from 'chalk';
+
+// Dependencies to be split/grouped into separate chunks.
+// This is useful to reduce the main bundle size and have more
+// dedicated caching strategy for specific chunks.
+const manualChunks = {
+  'commercetools-uikit-icons': ['@commercetools-uikit/icons'],
+  moment: ['moment', 'moment-timezone'],
+  'app-shell': [
+    '@commercetools-frontend/application-shell',
+    '@commercetools-frontend/application-shell-connectors',
+  ],
+};
+
+const removeNonExistantDependencies = (
```

Comment:
This is just to make sure the packages we're defining in the `manualChunks` map actually exists in the application to build.
This is a safety measure so in case one of them are remove (eg: moment) the build does not fail but will only emit a warning about it.

Reply:
Thanks for the comment. I was fearful we'd remove something else but those we specify at first.